content/project-news/hermes-belusca-hired-full-time.md

```diff
@@ -0,0 +1,27 @@
+---
+title:       "Hermès BÉLUSCA - MAÏTO hired full-time to work on the ReactOS GUI Setup"
+author:      "GeoB99"
+date:        2023-09-28
+---
+
+Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès BÉLUSCA - MAÏTO to work full-time on the ReactOS GUI Setup for the next 3 months.
```

Comment:
You may want to use international casing for his name instead of French casing, but it's also up to Hermes :)